### PR TITLE
[WIP] handle ConsoleGeneric messages

### PR DESCRIPTION
**Pull request by linclark**
Here's the basic component framework (with lots of TODOs to mark where I need to figure something out). I haven't double checked the markup structure yet.

Some of these components may end up going away. I think we should wait until we have more of them converted before figuring out where to consolidate.

Everything in here is up for discussion, so let me know if you have any thoughts or questions.

### CODE REVIEW

**Comment by linclark** [comment]
I forgot to mention... wherever you see `function Name(props)`, that is a component. It's just the same as a component that is created using `class Name extends Component`. The only difference is that it can't have state or use the lifecycle methods. That pattern is called a [stateless functional component](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions).

**Comment by bgrins** [comment]
Merged, thanks!  I also dropped in some basic styling here: https://github.com/bgrins/console-output-experiment/commit/2e69a807174df633dc8b00a9e195da01c80d3c56

**Comment by bgrins** [comment]
Oh I should note that I think this may have broke adding messages through the input field.  That's fine - we may just want to emit an message with category="input" when input happens though.

File: js/components/Message/Message.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+function Message(props) {
+  const { message } = props.packet;
+
+  const attributes = {
+    className: "message"
+    // @TODO Determine if category, severity and filter really need to be
+    // attributes here, or if they are just used for the filter actions.
+  };
+  // @TODO Under what circumstances is this true?
```
Comment (bgrins) [inline js/components/Message/Message.js]: This happens when a message is added from jsterm (not console api)

**Pull request merged**